scripts/build_wikipedia_dataset.py

```diff
@@ -0,0 +1,78 @@
+from datasets import list_datasets, load_dataset
+import jsonlines
+
```

Comment:
Don't we also need this?

```suggestion
import re
```